<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageScope</class>
 <widget class="QWidget" name="PageScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCustomPlot" name="ScopePlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Initial Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="I0">
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Step Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="I1">
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Step">
       <property name="text">
        <string>Step Current</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>On Fault</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CH1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Ch1source"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CH2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Ch2source"/>
     </item>
     <item>
      <widget class="QLabel" name="start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Samples</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="samples">
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="end">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Pretrigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pretrigger">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Divider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="divider">
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Run">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
